components/speedreader/speedreader_service.cc

```diff
@@ -72,7 +73,10 @@ void SpeedreaderService::RegisterProfilePrefs(PrefRegistrySimple* registry) {
       base::CommandLine::ForCurrentProcess()->HasSwitch(kCollectSwitch);
 #endif
 
-  registry->RegisterBooleanPref(kSpeedreaderPrefEnabled, enabled_by_deault);
+  registry->RegisterBooleanPref(kSpeedreaderPrefFeatureEnabled, true);
+  registry->RegisterBooleanPref(kSpeedreaderPrefEnabledForAllSites,
+                                enabled_by_deault);
```

Comment:
nit: `enabled_by_default`

Reply:
```suggestion
                                enabled_by_default);
```

Reply:
(and then there's the lines directly above this where it's defined / used once)